draft-ietf-homenet-front-end-naming-delegation.txt

```diff
@@ -741,13 +760,12 @@ Internet-Draft                public-names                     July 2021
 
    There are multiple ways this information could be exchanged between
    the HNA and the DM.  This specification defines a mechanism that re-
-   use the DNS exchanges format.  The intention is to reuse standard
-   libraries especially to check the format of the exchanged fields as
-   well as to minimize the additional libraries needed for the HNA.  The
-   re-use of DNS exchanges achieves these goals.  Note that while
-   information is provided using DNS exchanges, the exchanged
-   information is not expected to be set in any zone file, instead this
-   information is uses as commands between the HNA and the DM.
+   use the DNS exchanges format, while the exchange in itself is not a
+   DNS exchange involved in any any DNS operations such as DNS
+   resolution for example.  Note that while information is provided
+   using DNS exchanges, the exchanged information is not expected to be
+   set in any zone file, instead this information is uses as commands
```

Comment:
The suggested change was done. What we said with Dynamic update is that since it is not a centralized solution it is hard to control - as every devices do wat they want. 
The DNS exchange can be logged by the DM or the HNA.